homeassistant/components/hassio/strings.json

```diff
@@ -413,5 +413,11 @@
         }
       }
     }
+  },
+  "config": {
+    "step": { "system": {} },
```

Comment:
this is to make hassfest happy
Integration hassio:
* [ERROR] [TRANSLATIONS] Invalid strings.json: required key not provided @ data['config']['step']. Got None
* ```